# 強制速度更新GET
---
uint32 Flg
float64 Vx
float64 Vy
float64 Va

